IFC4x3/Templates/Partial Templates/Geometry/Curve Segment Geometry/Viennese Bend Transition Segment/DocTemplateDefinition.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Viennese_Bend_Transition_Segment_18qsFqvgTB0ftThvj4_r3E" Name="Viennese Bend Transition Segment" UniqueId="48d363f4-e6a7-4b02-9ddd-af9b44fb50ce" Type="IfcCurveSegment">
+	<Rules>
+		<DocModelRuleAttribute Name="ParentCurve">
+			<Rules>
+				<DocModelRuleEntity Name="IfcSeventhOrderPolynomialSpiral">
+					<Rules>
+						<DocModelRuleAttribute Name="ConstantTerm">
```

Comment:
See https://github.com/bSI-InfraRoom/IFC-Specification/pull/129#issuecomment-873583205. Committed (ad585d9296421a44652c5510af48f54e2b6ee1a1) a correction proposal.

Reply:
@czapplitec please look at these amends.

Reply:
Yes, I agree on this change